### PR TITLE
Csp 1352 email confirmation to add employer and set permissions

### PR DESCRIPTION
**Pull request by MarkFCain**

### CODE REVIEW

File: src/SFA.DAS.Provider.PR.Web/Views/AddEmployer/AddPermissionsAndEmployerSent.cshtml
Comment (DevenKShah) [inline src/SFA.DAS.Provider.PR.Web/Views/AddEmployer/AddPermissionsAndEmployerSent.cshtml:1]: This should be renamed to AddEmployerConfirmation

File: src/SFA.DAS.Provider.PR.Web/Controllers/AddEmployer/AddPermissionsAndEmployerController.cs [hunk 1]
```diff
@@ -64,14 +61,70 @@ public IActionResult Index([FromRoute] int ukprn, AddPermissionsAndEmployerViewM
 
         _sessionService.Set(sessionModel);
 
-        return RedirectToRoute(RouteNames.AddPermissionsAndEmployer, new { ukprn });
+        return RedirectToRoute(RouteNames.AddEmployerAndPermissionsSent, new { ukprn });
     }
 
+    [HttpGet]
+    [Route("/{ukprn}/addEmployer/addPermissions/sent", Name = RouteNames.AddEmployerAndPermissionsSent)]
```
Comment (DevenKShah) [inline src/SFA.DAS.Provider.PR.Web/Controllers/AddEmployer/AddPermissionsAndEmployerController.cs:68]: Rename the route to AddEmployerConfirmation

File: src/SFA.DAS.Provider.PR.Web/Controllers/AddEmployer/AddPermissionsAndEmployerController.cs [hunk 2]
```diff
@@ -2,21 +2,26 @@
...
 using SFA.DAS.Provider.PR.Web.Infrastructure.Services;
 using SFA.DAS.Provider.PR.Web.Models.AddEmployer;
 using SFA.DAS.Provider.PR.Web.Models.Session;
+using SFA.DAS.Provider.PR.Web.Services;
 
 namespace SFA.DAS.Provider.PR.Web.Controllers.AddEmployer;
 
 [Authorize(Policy = nameof(PolicyNames.HasContributorOrAbovePermission))]
 
 [Route("/{ukprn}/addEmployer/addPermissions", Name = RouteNames.AddPermissionsAndEmployer)]
```
Comment (DevenKShah) [inline src/SFA.DAS.Provider.PR.Web/Controllers/AddEmployer/AddPermissionsAndEmployerController.cs:20]: The user action does not add permission, it requests it. I think the route `/ukprn/addemployer` is sufficient as it implicitly then results in requesting permissions.

File: src/SFA.DAS.Provider.PR.Web/Views/AddEmployer/AddPermissionsAndEmployerSent.cshtml
Comment (MarkFCain) [reply src/SFA.DAS.Provider.PR.Web/Views/AddEmployer/AddPermissionsAndEmployerSent.cshtml:1]: ok

File: src/SFA.DAS.Provider.PR.Web/Controllers/AddEmployer/AddPermissionsAndEmployerController.cs (hunk 1 above)
Comment (MarkFCain) [reply src/SFA.DAS.Provider.PR.Web/Controllers/AddEmployer/AddPermissionsAndEmployerController.cs:68]: ok

File: src/SFA.DAS.Provider.PR.Web/Controllers/AddEmployer/AddPermissionsAndEmployerController.cs (hunk 2 above)
Comment (MarkFCain) [reply src/SFA.DAS.Provider.PR.Web/Controllers/AddEmployer/AddPermissionsAndEmployerController.cs:20]: ok

**Review by shaun-downey-education — APPROVED** [review]

**Pull request merged**